celestia/Blockdaemon.txt

```diff
@@ -0,0 +1,62 @@
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+        "description": {
+          "moniker": "Blockdaemon",
+          "identity": "35892AC552B8A0A8",
+          "website": "blockdaemon.com",
+          "security_contact": "support@blockdaemon.com",
+          "details": ""
+        },
+        "commission": {
+          "rate": "0.100000000000000000",
+          "max_rate": "0.200000000000000000",
+          "max_change_rate": "0.010000000000000000"
+        },
+        "min_self_delegation": "1",
+        "delegator_address": "celestia1evu846d408jdz5ju4883gtq9pnjvs8hzq8z2c9",
+        "validator_address": "celestiavaloper1evu846d408jdz5ju4883gtq9pnjvs8hz9cqnwr",
+        "pubkey": {
+          "@type": "/cosmos.crypto.ed25519.PubKey",
+          "key": "xwbuyz6bUNd+Ef79w0ysCU4qgDru1OZTn/yHnPaw5JE="
+        },
+        "value": {
+          "denom": "utia",
+          "amount": "66400000000"
+        }
```

Comment:
I think this was added by accident @JackPMallon its not the end of the world though we can just delete this file